athletics.lic

```diff
@@ -47,7 +47,7 @@ class Athletics
     elsif @swimming_target
       swim_loop(@swimming_target['rooms'])
     elsif @climbing_target
-      climb_practice(@climbing_target['id'], @climbing_target['target'], @climbing_target['hide'])
+      climb_practice(@climbing_target['id'], @climbing_target['target'], @climbing_target['target2'], @climbing_target['hide'])
```

Comment:
If anything here, I'd consider making `target` an array and then doing the above for each of them. A good thing to consider is why this is being done. Is it stopping too early I take it? So, it climbs for 15 ranks but not 30? Something like that?